From c245e87fad249622b1974dd64f3e497653269ee6 Mon Sep 17 00:00:00 2001
From: Parinay Kondekar <parinay.kondekar@seagate.com>
Date: Thu, 6 Jul 2017 08:48:00 +0530
Subject: [PATCH] LU-7653 tests: replay-single/110f fails for mdts on same MDS

replay-single/110f fails when multiple mdts are present on same MDS
node. Changing/reversing the failover order of mdts in the script
replay-singl/110f helps fix this failure in such setup.

Signed-off-by: Yang Sheng <yang.sheng@intel.com>
Signed-off-by: Parinay Kondekar <parinay.kondekar@seagate.com>
Change-Id: I8b528c422f0a90dd3174a5ff6c7f8d938ed68a05
Reviewed-on: https://review.whamcloud.com/27940
Tested-by: Jenkins
Tested-by: Maloo <hpdd-maloo@intel.com>
Reviewed-by: Jian Yu <jian.yu@intel.com>
Reviewed-by: Oleg Drokin <oleg.drokin@intel.com>
---
 lustre/tests/replay-single.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lustre/tests/replay-single.sh b/lustre/tests/replay-single.sh
index 2c85dae96e..0357ee4949 100755
--- a/lustre/tests/replay-single.sh
+++ b/lustre/tests/replay-single.sh
@@ -3877,7 +3877,7 @@ test_110f() {
 	replay_barrier mds1
 	replay_barrier mds2
 	$LFS mkdir -i1 -c$MDSCOUNT $DIR/$tdir/striped_dir
-	fail mds1,mds2
+	fail mds2,mds1
 
 	check_striped_dir_110 || error "check striped_dir failed"
 
-- 
2.17.1

